SimG4Core/Application/src/LowEnergyFastSimModel.cc

```diff
@@ -68,7 +68,7 @@ void LowEnergyFastSimModel::DoIt(const G4FastTrack& fastTrack, G4FastStep& fastS
   fastStep.KillPrimaryTrack();
   fastStep.SetPrimaryTrackPathLength(0.0);
   auto track = fastTrack.GetPrimaryTrack();
-  G4double energy = track->GetKineticEnergy();
+  G4double energy = track->GetKineticEnergy() * 1.015;
```

Comment:
Where this factor 1.015 comes from?
Define at least a constexpr value which defines it?